packages/@dcl/inspector/src/lib/babylon/decentraland/EcsEntity.ts

```diff
@@ -41,7 +41,7 @@ export class EcsEntity extends BABYLON.TransformNode {
   ecsComponentValues: EcsComponents = {}
 
   constructor(public entityId: Entity, public context: WeakRef<SceneContext>, public scene: BABYLON.Scene) {
-    super(`ecs-${entityId.toString(16)}`, scene)
+    super(`ecs-${entityId}`, scene)
```

Comment:
This was not part of the fix but it makes debugging easier to match the ID of the entity in the SDK with the ID of the node in babylon, i dunno why it was converted to hex 🤷